templates/home.html

```diff
@@ -51,15 +51,29 @@ <h4>
               <span class="text-muted">{{ msg.timestamp.strftime('%d %B %Y') }}</span>
               <p>{{ msg.text }}</p>
             </div>
-            <form method="POST" action="/users/add_like/{{ msg.id }}" id="messages-form">
-              <button class="
-                btn 
-                btn-sm 
-                {{'btn-primary' if msg.id in likes else 'btn-secondary'}}"
-              >
-                <i class="fa fa-thumbs-up"></i> 
-              </button>
-            </form>
+            {% if msg.user_id != g.user.id %}
```

Comment:
Hard to test logic inside a template -> Move any conditional into your route and pass those variables into the template. Example: `is_correct_user = msg.user_id == g.user.id`